packages/client/src/views/glsp-svg-views.tsx

```diff
@@ -0,0 +1,109 @@
+/********************************************************************************
+ * Copyright (c) 2024 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import {
+    ATTR_BBOX_ELEMENT,
+    BoundsAware,
+    Diamond,
+    Dimension,
+    GNode,
+    GPort,
+    GShapeElement,
+    Hoverable,
+    IViewArgs,
+    Point,
+    RenderingContext,
+    Selectable,
+    ShapeView,
+    svg
+} from '@eclipse-glsp/sprotty';
+import { injectable } from 'inversify';
+import { VNode } from 'snabbdom';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const JSX = { createElement: svg };
+
+@injectable()
+export class GDiamondNodeView extends ShapeView {
```

Comment:
I guess we should deprecate the `DiamondNodeView` and `CircularNodeView`  classes provided by sprotty and mention that the corresponding GView should be used instead.

Reply:
Good point!